MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -0,0 +1,175 @@
+                                                            HeaderText="Ship Country" />
+                            </sfgrid:SfDataGrid.Columns>
+                        </sfgrid:SfDataGrid>
+                    </pulltoRefresh:SfPullToRefresh.PullableContent>
+                </pulltoRefresh:SfPullToRefresh>
+            </Grid>
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="PullToRefreshTemplateBehavior.cs" %}
+
+    using Syncfusion.Maui.DataGrid;
+    using Syncfusion.Maui.PullToRefresh;
+
+    namespace PullToRefreshTemplate
+    {
+        public class PullToRefreshTemplateBehavior : Behavior<ContentPage>
+        {
+            private OrderInfoRepository? viewModel;
+            private Syncfusion.Maui.PullToRefresh.SfPullToRefresh? pullToRefresh;
+            private SfDataGrid? dataGrid;
+            protected override void OnAttachedTo(ContentPage bindable)
+            {
+                this.viewModel = new OrderInfoRepository();
+                bindable.BindingContext = this.viewModel;
+                this.pullToRefresh = bindable.FindByName<Syncfusion.Maui.PullToRefresh.SfPullToRefresh>("pullToRefresh");
+                this.dataGrid = bindable.FindByName<SfDataGrid>("dataGrid");
+                this.dataGrid.ItemsSource = this.viewModel.OrderInfoCollection;
+                this.pullToRefresh.Refreshing += this.PullToRefresh_Refreshing;
+                base.OnAttachedTo(bindable);
+            }
+
+            private async void PullToRefresh_Refreshing(object? sender, EventArgs e)
+            {
+                this.pullToRefresh!.IsRefreshing = true;
+                await Task.Delay(new TimeSpan(0, 0, 3));
+                this.viewModel!.ItemsSourceRefresh();
+                this.pullToRefresh.IsRefreshing = false;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## TransitionMode
+
+The [TransitionMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PullToRefresh.SfPullToRefresh.html#Syncfusion_Maui_PullToRefresh_SfPullToRefresh_TransitionMode) property specifies the mode of the animations. It has the following two modes:
+
+* [SlideOnTop](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PullToRefresh.PullToRefreshTransitionType.html#Syncfusion_Maui_PullToRefresh_PullToRefreshTransitionType_SlideOnTop)
+* [Push](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PullToRefresh.PullToRefreshTransitionType.html#Syncfusion_Maui_PullToRefresh_PullToRefreshTransitionType_Push)
+
+The default transition is `SlideOnTop` that draws the RefreshView on top of the `PullableContent`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh" 
+                                TransitionMode="SlideOnTop" />
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    pullToRefresh.TransitionMode = PullToRefreshTransitionType.SlideOnTop;
```

Comment:
Xaml code alone is enough.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/4a5ef285-8d49-4a11-9b64-8479bd9c8621)
Had removed the .cs code.